cfe_internal/update/update_policy.cf

```diff
@@ -597,4 +601,82 @@ body delete u_tidy
       rmdirs   => "true";
 }
 
+bundle agent modules_presence
+# @brief Render vendored and user provided modules from $(sys.inputdir) to $(sys.workdir)
+#
+# @description This bundle manages the contents of $(sys.workdir)/modules by
+#              first dealing with package module scripts.
+#              Preference is given to user provided package module scripts in
+#              modules/packages directory. If a module there matches a mustache
+#              template in modules/packages/vendored directory then the user
+#              provided module will be installed.
+#
+#              e.g. modules/packages/apt_get takes precedence over modules/packages/apt_get.mustache
+#
+#              Any other files in the modules directory will be promised to
+#              be updated in $(sys.workdir)/modules, including any sub-directories.
+{
+  vars:
+    "vendored_dir" string => "$(this.promise_dirname)/../../modules/packages/vendored/";
+    "override_dir" string => "$(this.promise_dirname)/../../modules/packages/";
```

Comment:
plus same for everyone else. because then they wont report to the enterprise hub by default and since we probably dont need that, let's conserve the bandwidth.